test/logflare/backends/postgres_adaptor_test.exs

```diff
@@ -131,22 +126,6 @@ defmodule Logflare.Backends.Adaptor.PostgresAdaptorTest do
   end
 
   describe "repo module" do
-    test "connected?/1 true when repo is connected", %{backend: backend} do
```

Comment:
can we update the tests to cover the SharedRepo's conencted?/1 instead? most of the sharedrepo functions aren't covered with tests.

Reply:
I wanted to remove `SharedRepo.connected?/1` as well, as it is not needed anywhere in the code.

Reply:
Gotcha, let's remove it within this PR too then